src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/configuration/SwaggerConfig.java

```diff
@@ -39,12 +42,15 @@ ApiInfo apiInfo() {
   
   @Bean
   public Docket customImplementation(){
+	  
+	  // Note: Enable of service based on environment name. 
+	  
       return new Docket(DocumentationType.SWAGGER_2)
               .select()
                   .apis(RequestHandlerSelectors.basePackage("ca.bc.gov.open.pssg.rsbc.dps.paymentservice"))
                   .paths(PathSelectors.any())
                   .build()
-              .apiInfo(apiInfo());
+              .apiInfo(apiInfo())
+              .enable(!environment.toUpperCase().contains("PROD"));
```

Comment:
could we use something like `swagger.ui.enable`= true/false or `spring.profiles.active` ?